tests/screenshot_test.py

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import io
+from pathlib import Path
+
+from PIL import Image
+
+import webtraversallibrary as wtl
+from tests.graphics_test import equal_images
+from webtraversallibrary.screenshot import Screenshot
+
+ORIGINAL_DIR = Path("tests/data/")
+
+
+class FakeWebdriver:
```

Comment:
Very minor comment but maybe MockWebdriver would be a better name?

Reply:
Renamed! :)